src/terminator/mod.rs

```diff
@@ -853,12 +855,50 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
             "sysconf" => {
                 let name = self.value_to_primval(args[0], usize)?.to_u64()?;
                 trace!("sysconf() called with name {}", name);
-                let result = match name {
-                    30 => PrimVal::Bytes(4096), // _SC_PAGESIZE
-                    70 => PrimVal::from_i128(-1), // _SC_GETPW_R_SIZE_MAX
-                    _ => return Err(EvalError::Unimplemented(format!("Unimplemented sysconf name: {}", name)))
-                };
-                self.write_primval(dest, result, dest_ty)?;
+                // cache the sysconf integers
+                let paths = &[
+                    (&["libc", "_SC_PAGESIZE"], PrimVal::Bytes(4096)),
+                    (&["libc", "_SC_GETPW_R_SIZE_MAX"], PrimVal::from_i128(-1)),
```

Comment:
Yea I'll add a comment.

Uhm... can we just define 4096 as the page size of miri? idk, this is funky business anyway. I don't have a strong opinion.